pkg/kudoctl/cmd/get/get_test.go

```diff
@@ -25,11 +27,7 @@ func TestValidate(t *testing.T) {
 
 	for _, tt := range tests {
 		err := validate(tt.arg)
-		if err != nil {
-			if err.Error() != tt.err {
-				t.Errorf("Expecting error message '%s' but got '%s'", tt.err, err)
-			}
-		}
+		assert.ErrorContains(t, err, tt.err)
```

Comment:
Why not use [assert.EqualError](https://godoc.org/github.com/stretchr/testify/assert#EqualError) of `testify` instead of including an additional assertion library?

Reply:
Oh, I see. Would be great if we could converge on a single lib for assertions, but that isn't something for this PR.

Reply:
maybe because Contains vs Equals? I kind of like asserting on just part of error messages. Btw. I did not know about any of these asserts so thatnks 👍